testing/trino-tests/src/test/java/io/trino/execution/TestSetSessionAuthorization.java

```diff
@@ -48,21 +46,11 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(TEST_SESSION)
-                .setSystemAccessControl(newFileBasedSystemAccessControl("set_session_authorization_permissions.json"))
+                .setSystemAccessControl("file", Map.of("security.config-file", getResourcePath("set_session_authorization_permissions.json")))
```

Comment:
nit:put args in separate lines

Reply:
i chose one line because it's simple and short. can it stay?

Reply:
a bit too many commas for me. No worries (marked as "nit")